content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Changing the container runtime from Docker Engine to containerd"
+weight: 8
+content_type: task 
+---
+This task outlines the steps needed to update your container runtime to containerd from Docker. It is applicable for cluster operators running Kubernetes 1.23 or earlier. 
+
+## Prerequisites 
+1. Install containerd. For more information see, [containerd's installation documentation](https://containerd.io/docs/getting-started/)
+
+
+2. Drain Node 
+   ```
+   # replace <node-to-drain> with the name of your node you are draining
+   kubectl drain <node-to-drain> --ignore-daemonsets
+   ```
+3. Stop the Docker daemon
+   ```
+   systemctl stop kubelet
+   systemctl stop docker
+   ```
+
+2.  Start containerd 
+
+## Migrate a Linux node {#migration-linux}
```

Comment:
I'd change this to something like:

\## Install containerd

To begin migrating your runtime, download and install containerd on your system.

\### Migrate a Linux node

*instructions*

\### Migrate a Windows node

*instructions*